pkgs/tools/misc/kak-lsp/default.nix

```
@@ -2,16 +2,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "kak-lsp";
-  version = "8.0.0";
+  version = "9.0.0";
 
   src = fetchFromGitHub {
-    owner = "ul";
+    owner = pname;
     repo = pname;
     rev = "v${version}";
-    sha256 = "0nka51szivwhlfkimjiyzj67nxh75m784c28ass6ihlfax631w9m";
+    sha256 = "1wfv2fy5ga6kc51zka3pak0hq97csm2l11bz74w3n1hrf5q9nnf8";
   };
 
-  cargoSha256 = "174qy50m9487vv151vm8q6sby79dq3gbqjbz6h4326jwsc9wwi8c";
+  cargoSha256 = "0g67s6n45rxvv1q5s7x5ajh5n16p68bhlsrsjp46qamrraz63d68";
 
   buildInputs = lib.optional stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ];
```

```suggestion
  buildInputs = lib.optional stdenv.isDarwin [ Security ];
```

Please add Security to inputs and inherit it from top level ``inherit (darwin.apple_sdk.frameworks) Security``.

---

Updated in 35a2887. I'm only able to test this on NixOS, but let me know if this is not how you expected it.

---

I meant like this https://github.com/NixOS/nixpkgs/commit/a72353735c1ca6e157e257318f3cda4649ea645e

---

Thanks, updated in 25e2b72. Let me know if you like me to note this change in commit message as well, although I don't know what this change will be described so as to be useful in a commit message.